src/controls/PrimitiveArrayControl.tsx

```diff
@@ -76,7 +76,7 @@ export function PrimitiveArrayControl({
     <Form.Item label={label} required={required}>
       <Form.List name="names">
         {(fields, { add, remove }, { errors }) => {
-          fields.length === 0 && add()
+          fields.length > 0 ? fields.forEach(() => add()) : add()
```

Comment:
this creates a render loop...looking into a diff solution

Reply:
Yeah I noticed that